src/pages/settings/bank-accounts/components/ConnectAccounts.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Button } from '@invoiceninja/forms';
+import { useTranslation } from 'react-i18next';
+import { MdLink } from 'react-icons/md';
+
+const ConnectAccounts = () => {
```

Comment:
We don't use arrow function expressions in the app. Please stick to convention with normal `function $name`.

Reply:
> We don't use arrow function expressions in the app. Please stick to convention with normal `function $name`.

Changed